### PR TITLE
Lowercase to uppercase: sentencenav -> SentenceNav

### PR DESCRIPTION
**Pull request by josephsl**
Maintenance: changes add-on ID from sentencenav to SentenceNav.

### CODE REVIEW

File: addons/SentenceNav/2.12.0.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"displayName": "SentenceNav (legacy)",
 	"publisher": "Tony Malykh <anton.malykh@gmail.com>",
 	"description": "Sentence Navigation addon.\nSentenceNav is an NVDA add-on that allows you to read text by sentences, as opposed to by paragraphs or words.",
-	"addonId": "sentencenav",
+	"addonId": "SentenceNav",
```
Comment (seanbudd) [inline addons/SentenceNav/2.12.0.json:5]: why not `sentenceNav` with an initial lowercase s?

**Comment by josephsl** [comment]
Hi,

That's what Tony put it as add-on ID/manifest name field - I know, this is nonstandard for us. @mltony, any thoughts?

Thanks.

**Comment by mltony** [comment]
I think NVDA devs should define naming convention for add-on names. It makes no difference to me whetehr the first letter is upper or lower case. But good to have consistent naming strategy.

**Comment by josephsl** [comment]
Hi, at the moment add-on names (manifest name field/add-on ID on store) starts with lowercase, with the manifest summary field used to display add-on title to users. Hope this clarifies things.

**Comment by seanbudd** [comment]
@josephsl should this be merged or does @mltony want to change the addonID further?

**Comment by josephsl** [comment]
Hi,

If Tony does not object, I think it would be best to merge this with a note about add-on manifest name convention discussion later.

Thanks.

**Pull request merged**